.github/workflows/test_update_prerelease_version.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import json
+
+import freezegun
+import pytest
+import update_prerelease_version
+
+TEST_DATETIME = "2023-01-01 01:23:45"
+
+
+def create_package_json(directory, version):
+    """Create `package.json` in `directory` with a specified version of `version`."""
+    package_json = directory / "package.json"
+    package_json.write_text(json.dumps({"version": version}), encoding="utf-8")
```

Comment:
Any reason you aren't using real data to test against by reading the actual `package.json` and updating the version before you write out the temp `package.json`?

Reply:
We ca do that, no real reason.